egs/babel_multilang/s5/local/nnet3/run_common_langs.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+. ./cmd.sh
+set -e
+stage=1
+train_stage=-10
+generate_alignments=true # false if doing ctc training
+speed_perturb=true
+use_flp=false
+use_pitch=true
+pitch_conf=conf/pitch.conf
+voicing_conf=
+use_pitch_plp=false
+[ ! -f ./conf/common_vars.sh ] && echo 'the file conf/common_vars.sh does not exist!' && exit 1
+
+. conf/common_vars.sh || exit 1;
+
+[ -f local.conf ] && . ./local.conf
+
+. ./utils/parse_options.sh
+
+
+L=$1
+
+# perturbed data preparation
+train_set=train
+if [ "$speed_perturb" == "true" ]; then
+  if [ $stage -le 1 ]; then
+    #Although the nnet will be trained by high resolution data, we still have to perturbe the normal data to get the alignment
+    # _sp stands for speed-perturbed
+    for datadir in train; do
+      utils/perturb_data_dir_speed.sh 0.9 data/$L/${datadir} data/$L/temp1
```

Comment:
use the *perturb_3way.sh script. Please add volume perturbation, which can also be done using a script like utils/data/volume_perturb.sh. Please check for the exact names.